src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-size: 18px;
+}
+
+.search_form {
+  height: 70px;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #000;
+  line-height: 70px;
+  margin: 20px 0;
+}
+
+.search_form_little {
+  height: 42px;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #000;
+  line-height: 42px;
+}
+
+.field {
+  height: 19px;
+  width: 158px;
+  border: none;
+  outline: none;
+  font-size: 18px;
+}
+
+.field_little {
+  height: 19px;
+  width: 158px;
+  border: none;
+  outline: none;
```

Comment:
don't repeat yourself, creacte some classes with common styles for both forms, imgs, inputs, and than use modificator, to change styles which is different for the second